osu.Game.Tests/Visual/SongSelect/TestScenePlaySongSelect.cs

```diff
@@ -1055,6 +1056,24 @@ public void TestModOverlayToggling()
             AddUntilStep("mod overlay hidden", () => songSelect!.ModSelect.State.Value == Visibility.Hidden);
         }
 
+        [Test]
+        public void TestBeatmapOptionsButtonDisable()
+        {
+            createSongSelect();
+
+            addRulesetImportStep(0);
+
+            AddAssert("delete option enabled", () => !songSelect!.DeleteOptionButton.Disabled);
+            AddAssert("clear option enabled", () => !songSelect!.ClearOptionButton.Disabled);
+            AddAssert("edit option enabled", () => songSelect!.EditOptionButton?.Disabled == false);
```

Comment:
The null-forgiving operator reads badly especially when used for boolean expressions. We generally use `AsNonNull` instead.